workloads/templates/workload-pvcscale-script-cm.yml.j2

```diff
@@ -0,0 +1,367 @@
+        if token:
+            self.token = token
+        if resdir:
+            self.resdir=resdir
+
+        try:
+            if not os.path.exists(self.resdir):
+              os.mkdir(self.resdir)
+        except OSError:
+            print ("Directory creation:", self.resdir, "failed")
+        else:
+            print ("Directory:", self.resdir, "created succesfully")
+
+      def get_json(self, url):
+        headers = {"Authorization": 'Bearer %s' % self.token}
+        try:
+            r = requests.get('https://%s:%s%s' % (self.host, self.port, url),
+                             headers=headers,
+                             verify=False)
+            parsed_json = r.json()
+        except ValueError:
+            print ("Unable to authenticate against OCP master", self.host, "check is token correct")
+            sys.exit()
+        except ConnectionError as e:
+            print ("Unable to connect to OCP master", self.host, "check connection to master")
+            sys.exit()
+
+        return parsed_json
+
+    # get all PVC from particular namespace
+      def get_pvc(self, namespace=None):
+        if namespace:
+          self.namespace = ns
+        api_pvc = '%s/namespaces/%s/persistentvolumeclaims' % (self.base_api, self.namespace)
+        parsed_json = self.get_json(api_pvc)
+        pvc_claims = []
+
+        for item in parsed_json["items"]:
+            pvc_claims.append(item)
+            pvc_claims = sorted(pvc_claims, key=lambda k: k['metadata']['creationTimestamp'],reverse=False)
+
+            with open(self.resdir+"/""pvc_"+str(ns)+".json", "w") as allpvc:
+                json.dump(pvc_claims, allpvc, indent=4)
+
+        # sorted - get stuff properly printed
+
+        with open(self.resdir+"/""pvc_"+str(ns)+".csv", 'wb') as cvsout:
+            csv_out = csv.writer(cvsout)
+            csv_out.writerow(['PVC Name', 'PVC Size' 'PVC Create Time','PVC Create Time - TZ' 'PVC Namespace'])
+            for pvc in pvc_claims:
+                csv_out.writerow([pvc["metadata"]["name"], pvc['spec']['resources']['requests']['storage'],
+                                  int(time.mktime(time.strptime(pvc["metadata"]["creationTimestamp"], '%Y-%m-%dT%H:%M:%SZ'))),
+                                  pvc["metadata"]["creationTimestamp"],
+                                  pvc["metadata"]["namespace"]])
+
+      def get_pods(self, namespace=None):
+
+        if namespace:
+            self.namespace = ns
+
+        api_pods = '%s/namespaces/%s/pods' % (self.base_api, self.namespace)
+        parsed_json = self.get_json(api_pods)
+        pods = []
+
+        # sort pods based on creationTime
+        for item in parsed_json["items"]:
+            pods.append(item)
+            pods = sorted(pods, key=lambda k: k["status"]["containerStatuses"][0]["state"]["running"]["startedAt"],reverse=False)
+            with open(self.resdir+"/""pods_"+str(ns)+".json", "w") as allpods:
+                json.dump(pods,allpods, indent=4)
```

Comment:
`, allpods`?

Reply:
yes, json.dump stuff,

Reply:
I believe Jiri was referring to the spacing.  It is a nitpick but makes for cleaner and more readable code.

Reply:
yes, consistency, please, spacing

Reply:
fixed, :), thanks @jmencak